x-pack/plugins/entity_manager/server/lib/entity_client.ts

```diff
@@ -18,23 +18,24 @@ import { startTransforms } from './entities/start_transforms';
 import { findEntityDefinitionById, findEntityDefinitions } from './entities/find_entity_definition';
 import { uninstallEntityDefinition } from './entities/uninstall_entity_definition';
 import { EntityDefinitionNotFound } from './entities/errors/entity_not_found';
-
 import { stopTransforms } from './entities/stop_transforms';
 import { deleteIndices } from './entities/delete_index';
 import { EntityDefinitionWithState } from './entities/types';
 import { EntityDefinitionUpdateConflict } from './entities/errors/entity_definition_update_conflict';
-import { EntitySource, getEntityInstancesQuery } from './queries';
-import { mergeEntitiesList, runESQLQuery } from './queries/utils';
-import { UnknownEntityType } from './entities/errors/unknown_entity_type';
+import { EntityClient as EntityClient_v2 } from './v2/entity_client';
 
 export class EntityClient {
+  public v2: EntityClient_v2;
+
   constructor(
     private options: {
-      esClient: ElasticsearchClient;
+      clusterClient: IScopedClusterClient;
```

Comment:
we'll have to update other plugins' instantiation of EntityClient (security, not sure about eco)

Reply:
Done!